components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/main/java/org/wso2/carbon/identity/notification/sender/tenant/config/NotificationSenderManagementServiceImpl.java

```diff
@@ -155,40 +157,21 @@ public EmailSenderDTO addEmailSender(EmailSenderDTO emailSender) throws Notifica
     @Override
     public SMSSenderDTO addSMSSender(SMSSenderDTO smsSender) throws NotificationSenderManagementException {
 
-        // Set the default publisher name if name is not defined.
-        if (StringUtils.isEmpty(smsSender.getName())) {
-            smsSender.setName(DEFAULT_SMS_PUBLISHER);
+        Map<String, String> properties = smsSender.getProperties();
+        String channel;
+        if (!StringUtils.isEmpty(properties.get(CHANNEL_TYPE_PROPERTY))) {
+            channel = properties.get(CHANNEL_TYPE_PROPERTY);
+        } else {
+            channel = properties.get("default");
```

Comment:
`properties.get("default");` or just `channel = "default"`

Reply:
resolved in https://github.com/wso2-extensions/identity-event-handler-notification/commit/6feda46da33ad906dc43f6c786db85b7f0bce845